src/e2e/java/teammates/e2e/pageobjects/Browser.java

```diff
@@ -131,6 +131,11 @@ public void close() {
      * Visits the given URL.
      */
     public void goToUrl(String url) {
+        if (TestProperties.BROWSER.equals(TestProperties.BROWSER_CHROME)) {
+            // Recent chromedriver has bug in setting page load timeout, which can potentially cause infinitely long waits
```

Comment:
what is the version of chromedriver + chrome that I can use to reproduce the said bug above? Might be good to mention that in this comment as well

Reply:
I'm not really sure when the entire thing started, all I can tell is it is present in the latest.